tests/integration/eZ/API/RichTextFieldTypeIntegrationTest.php

```diff
@@ -643,7 +644,8 @@ public function testConvertRemoteObjectIdToObjectId($test, $expected): void
      */
     public function testExternalLinkStoringAfterUpdate(): void
```

Comment:
Given exceptions reported by `getUrlIdForLink`, PHPDoc for this method could look like this:  
```php
    /**
     * @throws \Doctrine\DBAL\Driver\Exception
     * @throws \Doctrine\DBAL\Exception
     * @throws \ErrorException
     * @throws \eZ\Publish\API\Repository\Exceptions\Exception
     */
```